Library/Homebrew/build.rb

```diff
@@ -17,6 +17,11 @@
 require "cmd/install"
 require "json/add/exception"
 require "utils/output"
+require "extend/pathname/write_mkpath_extension"
+
+class Pathname
+  prepend WriteMkpathExtension
+end
```

Comment:
I could even move this to a `Pathname.prepend WriteMkpathExtension` just before we actually call `Formula#install` if preferred, not sure which is better practice